src/Symfony/Component/Translation/Tests/Dumper/FileDumperTest.php

```diff
@@ -16,6 +16,20 @@
 
 class FileDumperTest extends \PHPUnit_Framework_TestCase
 {
+    public function testDumpWithFormatCallback()
+    {
+        $catalogue = new MessageCatalogue('en');
+        $catalogue->add(array('foo' => 'bar'));
+
+        $dumper = new ConcreteFileDumper();
+        $format = false;
+        $dumper->dump($catalogue, array('format_callback' => function ($content) use (&$format) {
+            $format = true;
```

Comment:
you should probably have an assertion on the content too, not just on the fact it got called